readthedocs/projects/forms.py

```diff
@@ -473,6 +473,10 @@ def clean_url(self):
             raise forms.ValidationError(
                 _('This field is required.')
             )
+        if 'https' not in url:
```

Comment:
Other options include:
```python
# option 1
if url.startswith('http://'):
    raise forms.ValidationError(
            _('Please provide an https URL.')
        )

# option 2
from urllib.parse import urlparse

url = urlparse(url)

if url.scheme.startswith('http://'):
    raise forms.ValidationError(
            _('Please provide an https URL.')
        )
```

Reply:
I think is better if we check for `^https?://` with a regex

Reply:
Probably better to use `urlparse` and check for the scheme than applying a regex over a user input.